src/createDocumentActionDuplicate.js

```diff
@@ -0,0 +1,57 @@
+import sanityClient from 'part:@sanity/base/client'
+import { CopyIcon } from '@sanity/icons'
+import { i18n as getI18n } from './i18n'
+import { useRouter } from '@sanity/base/router'
+import pluginConfig from 'config:@kaliber/sanity-plugin-duplicate'
+
+const client = sanityClient.withConfig({
+  apiVersion: '2022-04-06'
+})
+
+export function createDocumentActionDuplicate(documentSchemes) {
```

Comment:
Dit is niet nodig:
```suggestion
export function createDocumentActionDuplicate() {
```

Je kunt dit doen om het schema erbij te pakken:

```js
import schema from 'part:@sanity/base/schema'

...

schema.get(type).fields...
```